packages/algorand-js/test/src/interpreter/opcode-list.ts

```diff
@@ -412,4 +413,187 @@ describe("Teal Opcodes", function () {
       );
     });
   });
+
+  describe("Mod", function () {
+    const stack = new Stack<StackElem>();
+
+    it("should return correct modulo of two unit64", function () {
+      stack.push(BigInt("2"));
+      stack.push(BigInt("5"));
+      const op = new Mod();
+      op.execute(stack);
+
+      const top = stack.pop();
+      assert.equal(top, BigInt("1"));
```

Comment:
Please add tests for:
* (7, 7)


Can be in the same unit-test (same "it")

Reply:
ok